content/03.results.md

```diff
@@ -3,16 +3,14 @@
-Next, we created a Github analysis repository (https://github.com/AlexsLemonade/OpenPBTA-analysis) with continuous integration to ensure analysis reproducibility and a GitHub manuscript repository (https://github.com/AlexsLemonade/OpenPBTA-manuscript) with ManuBot [@url:https://manubot.org/ ; @doi:10.1371/journal.pcbi.1007128] integration to enable manuscript creation using Markdown within GitHub. 
+We then harnessed the benchmarking efforts of the Kids First Data Resource Center to develop a robust and reproducible data analysis workflow within the CAVATICA platform to perform the following primary somatic analyses: variant calling of single nucleotide variants (SNVs), copy number variants (CNVs), structural variants (SVs), and fusions (**Figure S1}** - red boxes and **STAR Methods**). 
+Next, we created a public Github analysis repository (https://github.com/AlexsLemonade/OpenPBTA-analysis) with continuous integration to ensure analysis reproducibility.
+We additionally created a GitHub manuscript repository (https://github.com/AlexsLemonade/OpenPBTA-manuscript) with ManuBot [@url:https://manubot.org/ ; @doi:10.1371/journal.pcbi.1007128] integration to enable manuscript creation using Markdown within GitHub.
 We maintained a data release folder on Amazon S3 containing merged files for each analysis, downloadable from the GitHub repository or open access CAVATICA project.  
 
-Stub for the process for contributing analytical code (**Figure {@fig:FigS1}** and **STAR Methods**) and to the manuscript.
-Also add a quick note about consensus SNV and CNV to reference **Figure {@fig:FigS2}A-F** here.
-
-![**Figure 1. Overview of the OpenPBTA Project.** ](images/main/Figure1.png){#fig:Fig1}
-<!-- TODO: Add figure 1 description. -->
+<!-- TODO: Stub for the process for contributing analytical code (**Figure {@fig:FigS1}** and **STAR Methods**) and to the manuscript. -->
 
+![**Figure 1. Overview of the OpenPBTA Project.** A, The Children's Brain Tumor Network and the Pacific Pediatric Neuro-oncology Consortium collected tumor samples from 943 patients. To date, 38 cell lines were created from tumor tissue and over 2000 specimens were sequenced (N = 1035 RNA-Seq, N = 940 WGS, and N = 32 WXS or Targeted panel). Data was harmonized by the Kids First Data Resource Center using an Amazon S3 framwork within CAVATICA. B, Processed data was released through the OpenPBTA GitHub project, through which contributors filed issues andd developed code for additional analyses within docker. Additional analysis details can be found in **Figure S1**. Simultaneously, this manuscript was written in GitHub using Manubot. All analysis and manuscript code was reviewed and continuously integrated using CircleCI. C, Stacked bar plot summary of the number of biospecimens per phase of therapy  per broad histology.](images/main/Figure1.png){#fig:Fig1}
```

Comment:
Same suggestions for a couple typos (like `framwork`) phrasing, but _not_ OPSL.

```suggestion
![**Figure 1. Overview of the OpenPBTA Project.** 
(A) The Children's Brain Tumor Network and the Pacific Pediatric Neuro-Oncology Consortium collected tumor samples from 943 patients. To date, 38 cell lines were created from tumor tissue and over 2000 specimens were sequenced (N = 1035 RNA-Seq, N = 940 WGS, and N = 32 WXS or Targeted panel). Data was harmonized by the Kids First Data Resource Center using an Amazon S3 framework within CAVATICA. (B) Processed data was released through the OpenPBTA GitHub project, specifically in the OpenPBTA-analysis repository, through which contributors filed issues and developed code for additional analyses within a Docker contain. Additional analysis details can be found in **Figure {@fig:FigS1}**.  Simultaneously, this manuscript was written in GitHub using Manubot in the accompanying OpenPBTA-manuscript repository. All analysis and manuscript code was reviewed by pediatric cancer experts and analytical code experts and continuously integrated using CircleCI. (C) Stacked bar plot summary of the number of biospecimens per phase of therapy per cancer group.](images/main/Figure1.png){#fig:Fig1}
```

Reply:
Ah I think I missed that because when it went to OSPL, it showed the entire thing as different, rather than the word. Are there typos other than "framework here"? because it went back to a new line. I hadn't used parentheses in any figure, hence not accepting those. Maybe we save that for copy edit? I also tend to bold those, but this isn't the right place anyway because I think this will all go away in the submission and what is the important figure legend would be in the administrative doc.

Reply:
I also unlinked S1, but I see you re-linked it. We are going to not link supplemental figures, right?

Reply:
ok, found it

Reply:
Sorry didn't mean to re-link! 
The other important typos were "andd" and capitalizing "O" in "Pacific Pediatric Neuro-Oncology Consortium"